sparkles/tests/test_review.py

```diff
@@ -629,7 +630,7 @@ def test_roll_options_for_not_allowed_pitch():
     roll = nominal_roll(ra, dec, date)
     att0 = Quat([ra, dec, roll])
 
-    att = apply_sun_pitch_yaw(att0, pitch=40)
+    att = apply_sun_pitch_yaw(att0, pitch=40, time=date)
```

Comment:
The new ska_sun brakes on the previous line of code here... which I think was wrong anyway.

Reply:
And this change is needed either way... though I think new ska_sun with position_accurate should probably handle the time=None sun_ra=None sun_dec=None case better than an astropy.time ValueError.

Reply:
This is a real bug which I have a fix for, PR coming.